imap_processing/tests/ccsds/test_excel_to_xtce.py

```diff
@@ -0,0 +1,60 @@
+"""Testing for xtce generator template."""
```

Comment:
Is this file the same as test_xtce_generator_template?

Reply:
Yes, with a few updates to satisfy the CLI changes.

Reply:
Great, that's what I assumed so I didn't give it a close review.